R/labelled.R

```diff
@@ -91,6 +91,7 @@ make_labelled <- function(supertbl, format_labels = NULL) {
   data_labs <- c( # nolint: object_usage_linter
```

Comment:
`labels` :)

Reply:
Gah!
👍

Reply:
I'll just go ahead and change all of the `*_labs` then, this was in place since `make_labelled()` was first written.